src/apis/modTools.es6.js

```diff
@@ -22,19 +28,46 @@ const approve = (apiOptions, fullname) => {
   return apiRequest(apiOptions, 'POST', 'api/approve', { body, type: 'form' });
 }
 
-const distinguish = (apiOptions, fullname, distinguishType) => {
-  // Distinguish a link or comment
-
+/**
+ * Distinguish a link or comment
+ * @function
+ * @param {Object} apiOptions
+ * @param {string} fullname The fullname of the target comment
+ * @param {DISTINGUISH_TYPES} distinguishType What type of distinguish is being set
+ * @param {?bool} [_sticky] For internal use by #setStickyComment
+ */
+const distinguish = (apiOptions, fullname, distinguishType, _sticky=null) => {
   const distinguishTypeMap = {
-    'moderator': 'yes',
-    '': 'no',
+    [DISTINGUISH_TYPES.MODERATOR]: 'yes',
+    [DISTINGUISH_TYPES.NONE]: 'no',
   };
 
   const body = {
-    how: distinguishTypeMap[distinguishType],
     id: fullname,
   };
-  return apiRequest(apiOptions, 'POST', 'api/distinguish', { body, type: 'form' });
+  const how = distinguishTypeMap[distinguishType] || distinguishType;
```

Comment:
since `admin` is defined in DISTINGUISH_TYPES, to support it we either need to define it in `distinguishTypeMap`, or just add a fallback to use the value directly (what the `|| distinguishType` is doing)